R/EnrichDepletTF.R

```diff
@@ -0,0 +1,417 @@
+#'                             quantile normalized and fetched by Dr. Boris Steipe
+#'                             \href{https://github.com/hyginn/BCB420.2019.ESA/blob/master/R/fetchData.R}{fetchData Github}
+#'                             The matrix is produced by the function sysExProf() and the
+#'                             row dimentions are set by the length of the vector of the
+#'                             queried genes that was fed into sysExProf().
+#' @param nUpMost (numeric / boolean) Either "FALSE" to returns all significantly
+#'                                    correlated genes, a numeric or vector of 1L length,
+#'                                    describing the amount of upmost correlated pairs of
+#'                                    genes to return by the function. The default is set
+#'                                    to FALSE.
+#' @param direction (char)     A character vector of 1L length of either "Positive" or
+#'                             "Negative". "Positive" returns pairs of genes that are
+#'                             signigicantly positively correlated. "Negative" returns
+#'                             pairs of genes that are signigicantly negatively correlated.
+#'                             The default is set to "Positive".
+#' @param multipleTests (char) A character vector of 1L length of either "Bonferroni" or
+#'                             "BH" (Benjamini-Hochberg), to set the multiple tests
+#'                             correction approach. The default is set to "Bonferroni".
+#' @param alpha (numeric)      a numeric or vector of 1L length, to set the value of alpha
+#'                             for deciding on the cutoff for significant results. The
+#'                             Default value is 0.05.
+#' @return (char vector)       A character vector of unique correlated of genes (HGNC
+#'                             symbols). Although the number of pairs of genes that
+#'                             constitutes the vector is set by the paramenter "nUpMost",
+#'                             the length of the vector may vary and won't always be
+#'                             2 * "nUpMost", because duplications are deleted so that each
+#'                             gene appears only once.
+#'
+#' @family <optional description of family>
+#'
+#' @author \href{https://orcid.org/0000-0002-9478-5974}{Sapir Labes} (aut)
+#'
+#' @seealso \code{\link{cor.test}} Test for association between paired samples, using one
+#'                                 of Pearson's product moment correlation coefficient,
+#'                                 Kendall's tau or Spearman's rho.
+#'
+#' @examples
+#' # Vector of the 12 up most positively correlated pairs of genes of "SLIGR"
+#' # system.
+#' GeneSym <- SyDBgetSysSymbols(fetchData("SysDB"),"SLIGR")[[1]]
+#' exProfs <- sysExProf(GeneSym = GeneSym)
+#' sysUpCorGenes(exProfS = exProfs, nUpMost = 12)
+#'
+
```

Comment:
missing `@export`

Reply:
About that,
Didn't you say in class that only the exported function needs to have "@export" tag?